content-src/experiments/universal-search.yaml

```diff
@@ -91,6 +91,68 @@ contributors:
 installation_count: 39943
 created: '2016-04-28T03:57:32.270681Z'
 modified: '2016-07-20T15:42:28.873321Z'
+completed: '2016-11-30T16:00:00.000000Z'
+graduation_report: >
+  <img src="/static/images/experiments/universal-search/experiments_experimentdetail/6/d/6d0ff50eff9223f0d0cc58c5aa5d0c81_image_1462827811_0995.jpg" alt="universal search preview" />
```

Comment:
Is this image going to exist when this goes out?  (I don't see it in this PR)  Same with the one below